src/org/thoughtcrime/securesms/components/ThumbnailView.java

```diff
@@ -312,15 +319,23 @@ public void clear(GlideRequests glideRequests) {
     slide = null;
   }
 
+  public void setFit(@NonNull BitmapTransformation fit) {
+    this.fit = fit;
+  }
+
   private GlideRequest applySizing(@NonNull GlideRequest request, @NonNull BitmapTransformation fitting) {
     int[] size = new int[2];
     fillTargetDimensions(size, dimens, bounds);
     if (size[WIDTH] == 0 && size[HEIGHT] == 0) {
       size[WIDTH]  = getDefaultWidth();
       size[HEIGHT] = getDefaultHeight();
     }
-    return request.override(size[WIDTH], size[HEIGHT])
-                  .transforms(fitting, new RoundedCorners(radius));
+    request = request.override(size[WIDTH], size[HEIGHT]);
+    if (radius > 0) {
+	return request.transforms(fitting, new RoundedCorners(radius));
+    } else {
+	return request.transforms(fitting);
```

Comment:
```suggestion
      return request.transforms(fitting, new RoundedCorners(radius));
    } else {
      return request.transforms(fitting);
```
to make the indentation show up in Android Studio